tom_hermes/hermes.py

```diff
@@ -92,9 +86,9 @@ class HermesNonLocalizedEventAlert:
     id: int # used for cache identification and retrieval 
     event_id: str
     nonlocalizedevent_id: int # PK of tom_nonlocalizedevent.model.NonLocalizedEvent (if exists)
-    title: str
```

Comment:
Removed title because it doesn't hold much meaning with igwn alerts - I make up the title as event_id + sequence type, which are both already shown in the table.